stdlib/2and3/codecs.pyi

```diff
@@ -13,10 +13,13 @@ import types
 _Decoded = Text
 _Encoded = bytes
 
+class _Encoder:
```

Comment:
You need to make these inherit from Protocol, otherwise only these exact classes will be accepted.

Reply:
Done